modules/ossm-release-notes-istio-traffic-management.adoc

```diff
@@ -0,0 +1,42 @@
+////
+Module included in the following assemblies:
+* service-mesh-docs-main/ossm-release-notes-support-tables-assembly.adoc
+////
+
+//Post GA: might be worth considering putting tables into their own modules. Might easier to maintain. Warrants further discussion post GA.
+
+:_mod-docs-content-type: REFERENCE
+[id="istio-traffic-management_{context}"]
+= Istio traffic management
+
+[cols="1,1"]
+|===
+| Feature | Status
+
+| Protocols: HTTP1.1/HTTP2/HTTPS/gRPC/TCP/TLS
+| GA
+
+| Traffic Control: label/content based routing, traffic shifting
```

Comment:
For completeness (after talking to a customer), can we add an entry:
Feature: VirtualService, DestinationRule and ServiceEntry
Status: GA